functions/rename-requirement-and-slot-name.js

```diff
@@ -0,0 +1,109 @@
+// General script to rename the requirement or slot name in Firestore if it
+// has been changed in the requirements data.
+const admin = require('firebase-admin');
+const fs = require('fs');
+const serviceAccount = require('./service-account.json');
+
+admin.initializeApp({
+  credential: admin.credential.cert(serviceAccount),
+  databaseURL: 'https://cornelldti-courseplan-dev.firebaseio.com',
+});
+
+const db = admin.firestore();
+const userOverriddenReqsCollection = db.collection('user-overridden-reqs');
```

Comment:
To be consistent with the naming scheme (i.e. `'user-selectable-requirements'`), can we do `'user-overriden-requirements'`?